src/code/entities/bank6.asm

```diff
@@ -89,7 +89,7 @@ SetEntitySpriteVariantForDirection::
     jp   SetEntitySpriteVariant                   ; $645A: $C3 $0C $3B
 
 ; Should Link talk to Entity
-func_006_645D::
+CheckLinkInteractionWithEntity_06::
```

Comment:
I thought this was a variant of ShouldLinkTalkToEntity_**, but it actually does more thing, like checking for item usage ; so 👍